### DISCUSSION TITLE
Scrum Prep Milestone #1 - Third Week

### DISCUSSION THREAD

**Discussion by evnd04** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#167] [Weekly Retrospective]
This weeks scrum prep is a bit different, we would like an overview of the whole milestone and your work accomplished throught the milestone. 

As for me, i worked mostly on set ups for the point tracker which for some reason isnt running yet, also provided the task template for efficient and well redacted tasks, spent my tme reviewing pr's and approving tasks, also grading them based on description, scale and redaction quality. We held team lead meetings where me and @YanehRuiz assesed weekly progress also.

**Comment by Jjulianirizarry24** [comment]
I worked on setting up a proper base for the report map feature for the safe rum app. I achieved this through creating mock ups for the map, leading team meetings discussing the progress of the location management team, setting up the react environment and implementing the initial map view. Moving forward, I'd like to keep adding functionality to the feature on milestone #2 to have a working report map under the correct boundaries.

**Comment by lex939** [comment]
I completed my remaining open documentation issues, went to LM meetings to discuss documentation progress and report any important changes to the managers, and polished the document to make it ready for submission. 

The documentation for M1 is due by Friday, September 20 at 8pm (day of the scrum prep in class). I will be discussing with @evnd04 and @YanehRuiz throughout the day to make sure everything is cleared and ready before submission.

I'll be staying tuned to feedback from the professor to see how we can improve our proposal for the next submission. Looking forward to submitting our proposal for Milestone 1 and starting Milestone 2!

**Comment by JorgeRivera94** [comment]
Finished creating the design for report creation pages and their user flow for issue #12. Can be found in discussion #171. This involved freehand sketches and learning to use Figma tools to represent the final version of these designs.

**Comment by shenied-maldonado** [comment]
I personally focused on researching user authentication and session management services. We're still in discussions regarding that, but we're a step closer in thinking about what we want for SafeRUM. Other than that, I focused on making sure my team lifted off properly for our first milestone. Prioritizing UI designs and schematics, I led meetings, kept track of our logbook, and provided feedback for the User Authentication team. Our most notable accomplishments include the database schema design and our collection of uploaded wireframes for User Account Creation, User Profile Management, and User Login/Sign-Up. With that, we hope to finally implement our ideas during this coming milestone.

**Comment by NathBat09** [comment]
I worked on the database API for reports with CRUD operations and an initial design for report deletion.

**Comment by ghost** [comment]
Worked on documentation tasks, specifically #73 (2.1.1 Domain Rough Sketch) and #85 (2.3 Implementation). Will also finish #110 (Design for Editing Reports) by EOD.

**Comment by YanehRuiz** [comment]
Throughout the milestone I held and attended meetings with @evnd04. Made a task point grading document with information about the points, provided documentation references, helped with team division tasks and helped approve and review pr and issues.

**Comment by javier-rivera8** [comment]
This week, I attended meetings and completed my tasks, which included implementing credential logic, verification, forgot password functionality, editing the user profile, and working on the project presentation. I didn’t encounter any problems, and the components are running smoothly.

**Comment by CaribbeanCool** [comment]
In this milestone, I created a user flow diagram of the map page, wrote a one pager about which technology to use for implementing a Map view, and this week I worked on documentation.

**Comment by jonVargas1** [comment]
I worked on implementing a basic UI for report creation and began working on the design for viewing reports.

**Comment by Chosen-Juan1** [comment]
This week, I designed the admin communication's interface page and worked on the documentation.

**Comment by yadrielsamir** [comment]
Throughout the whole milestone I conducted meetings for the admin interface team where weekly progress was discussed and finished the profile page design and implementation. For this week I made the official figma page where all figma designs will be added in order to maintain familiarity between the app designs (issue #137).

**Comment by williammartinez10** [comment]
I focused on developing the UI components for the signup, login, and landing page as part of the User authentication system. Additionally, I have started researching how to implement the backend functionalities for User authentication.

**Comment by johnsantiago9** [comment]
During this milestone I work with the schematics of the user profile design and worked directly with the documentation of the project. For next milestone i will be working with the sign up and user account creation UI

**Comment by noel-vargas** [comment]
I created the user profile management tab in react native, UML diagrams and figma designs for account creation, logins in and forgot password wireframes

**Comment by joel-alvarado** [comment]
For Milestone #1, I had been assigned with working everything related to the initial database. I created scripts to initialize the tables, add dummy data to it and wrote a docker-compose configuration file so developers could spin up a container with the database with ease and allow data permanence after the container is shut down.

**Comment by Alejandro-DeJesus** [comment]
I worked on task #100 and completed it, for the milestone, also worked on task #23, but was unable to progress much since it carries a big scope while depending on many other tasks completion, suggesting task is scrapped and reworked for milestone 2.

**Comment by JuanJRivera** [comment]
I completed the documentation for several key sections related to the SafeRUM project. This included defining important terms and concepts in the 2.1.2 Terminology section, developing a detailed domain description in the 2.1.4 Narrative, outlining key interactions in 2.1.5 Events, Actions, and Behaviors, and creating function signatures for various actions in 2.1.6 Function Signatures. All these sections have been approved and are ready for the next steps in the project.

**Comment by JorgeC08** [comment]
I finished my documentation task and currently finishing the incorrect password handling. Is giiving me problems connecting to the database but I'm working on fixing it.

**Comment by RaquelQuinones** [comment]
Finished task #17 and worked with the password handling task.

**Comment by lex939** [comment]
The Scrum Prep for Friday, September 20, has started. The Weekly Retrospective for the second week of Milestone 1 is now closed. Please note that any replies after this comment will be considered as late replies.

**Comment by Rodal1** [comment]
Completed the admin settings page and also the edit profile page. Attended group meetings, wrote summary for each team meeting as well.
